### DISCUSSION TITLE
Is it only me who have much problem with implicit types for load functions, actions etc?

### DISCUSSION THREAD

**Discussion by einarpersson** [sveltejs/kit#10703] [General]
Hi! I have lately gotten so many errors where the implicit types for load functions etc. fail.

As a typical example, I just opened my project and browser around. In a ´+page.server.ts` I suddenly saw this:

![image](https://github.com/sveltejs/kit/assets/6043691/8fcd860e-cdd0-4f9f-8c7c-686d6cded1dc)

After reloading VSCode
![image](https://github.com/sveltejs/kit/assets/6043691/e6a8661c-143b-4a41-be48-86136427e49d)

However, this is happening **_all too often_** to the point that it is hindering my daily work. Last friday it felt as if I was not doing anything else than reloading and restarting VSCode/dev server, deleting .svelte-kit etc, and the errors just seem to be moving randomly.

Am I the only one experiencing this? What could be the cause? I have almost everything up-to-date (Sveltekit, VSCode, Vscode Extension) but have not upgrade to svelte4 yet. 

tsconfig.json
```json
{
	"extends": "./.svelte-kit/tsconfig.json",
	"compilerOptions": {
		"allowJs": true,
		"checkJs": true,
		"esModuleInterop": true,
		"forceConsistentCasingInFileNames": true,
		"resolveJsonModule": true,
		"skipLibCheck": true,
		"sourceMap": true,
		"strict": true,
		"noErrorTruncation": true
	}
	// Path aliases are handled by https://kit.svelte.dev/docs/configuration#alias
	//
	// If you want to overwrite includes/excludes, make sure to copy over the relevant includes/excludes
	// from the referenced tsconfig.json - TypeScript does not merge them in
}
```

package.json-deps
```json
"devDependencies": {
		"@iconify/svelte": "3.1.3",
		"@playwright/test": "1.34.3",
		"@sveltejs/adapter-vercel": "3.0.0",
		"@sveltejs/kit": "1.24.1",
		"@typescript-eslint/eslint-plugin": "6.6.0",
		"@typescript-eslint/parser": "6.6.0",
		"contentful": "10.2.3",
		"d3-dsv": "3.0.1",
		"eslint": "8.41.0",
		"eslint-config-prettier": "8.8.0",
		"eslint-plugin-svelte3": "4.0.0",
		"googleapis": "126.0.0",
		"mimetext": "3.0.16",
		"pocketbase-typegen": "1.1.12",
		"prettier": "2.8.8",
		"prettier-plugin-svelte": "2.10.1",
		"svelte": "3.59.1",
		"svelte-check": "3.5.1",
		"tslib": "2.6.2",
		"type-fest": "4.3.1",
		"typescript": "5.2.2",
		"vite": "4.3.9",
		"vitest": "0.31.2"
	},
	"type": "module",
	"dependencies": {
		"@picocss/pico": "1.5.10",
		"fast-deep-equal": "3.1.3",
		"pocketbase": "0.17.2",
		"pupa": "3.1.0",
		"svelte-multiselect": "8.6.2",
		"zod": "3.21.4",
		"zod-form-data": "2.0.1"
	}
```

**Comment by geoffrich** [comment]
This issue is being tracked in https://github.com/sveltejs/language-tools/issues/2053

**Reply by einarpersson** [reply]
Thank you, I had searched but it was hard to know exactly what to search for. Hence this question here in discussions. Thank you for pointing me in the right direction
